options.h

```diff
@@ -57,6 +57,9 @@ class RtSmithOptions : public AbstractP4cToolOptions {
 
     // The control plane API to use. Defaults to P4Runtime.
     std::string _controlPlaneApi = "P4RUNTIME";
+
+    // The type of the configuration to generate.
+    std::string _configType = "";
```

Comment:
This also does not seem to be needed.

Reply:
Sorry! I will remove all the dumps I made (due to some perhaps experimental thoughts a while ago that I forgot) before your reviews.